homeassistant/components/ai_speaker/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "ai_speaker",
+  "name": "AI-Speaker integration",
+  "documentation": "https://ai-speaker.com",
+  "dependencies": [],
```

Comment:
Please remove empty items.

Reply:
ok, done in commit @asystentka-jolka
improvements after first review
6214ef8